data_diff/diff_tables.py

```diff
@@ -120,25 +137,45 @@ def _get_stats(self) -> DiffStats:
         table2_count = self.info_tree.info.rowcounts[2]
         unchanged = table1_count - diff_by_sign["-"] - diff_by_sign["!"]
         diff_percent = 1 - unchanged / max(table1_count, table2_count)
+        if is_dbt:
+            return DiffStats(diff_by_sign, table1_count, table2_count, unchanged, diff_percent, extra_column_diffs)
+        else:
+            return DiffStats(diff_by_sign, table1_count, table2_count, unchanged, diff_percent, None)
 
-        return DiffStats(diff_by_sign, table1_count, table2_count, unchanged, diff_percent)
-
-    def get_stats_string(self):
+    def get_stats_string(self, is_dbt:bool = False):
 
         diff_stats = self._get_stats()
```

Comment:
`diff_stats = self._get_stats()`
this should move to the else so we don't run _get_stats() twice when is_dbt == True

Reply:
Does my latest update accomplish the same thing? Passing the `is_dbt` variable to `_get_stats`? https://github.com/datafold/data-diff/pull/381/files#diff-b8c2ad85fce41be1450cde985be83f28803c095ba5d3f0235fc916ed8f965e80R144